### DISCUSSION TITLE
How to copy a view?

### DISCUSSION THREAD

**Discussion by JrgMyr** [pawelsalawa/sqlitestudio#4168] [Q&A]
Is there a preferred way how to use a view to create a similar but different one?
I could think of a function "copy view" or a button "save as new view" in the editing pane.

Now, I open the original view, do Ctrl-A Ctrl-C, create a new view, insert the previous content with Ctrl-V and add a new name.
Possible, yes, but not very elegant.

Cheers, Jörg

**Comment by pawelsalawa** [comment]
Currently there is no such feature as "create similar view". I thought it's a very simple thing to just copy-paste the query of a view, thus I didn't consider such feature as useful. For a table this is much more complex, thus it's useful there.

**Reply by JrgMyr** [reply]
Understood.
It is just so that my analysis databases have way more queries than tables. :-(

A bit of "query management" support would come in handy. The ideal world would also timestamp "creation" and "last change" of every query to facilitate sorting them. Just like it can be done with files in a folder.

**Reply by pawelsalawa** [reply]
I think that #4065 would be something like you mention here.
